// html-validate/dist/es/browser.js
export { c as compatibilityCheck } from './core-browser.js';
export { A as Attribute, C as Config, a as ConfigError, b as ConfigLoader, h as DOMNode, d as DOMTokenList, i as DOMTree, D as DynamicValue, E as EventHandler, f as HtmlElement, H as HtmlValidate, n as MetaCopyableProperty, M as MetaTable, l as NestedError, N as NodeClosed, j as NodeType, x as Parser, u as Reporter, R as ResolvedConfig, o as Rule, k as SchemaValidationError, S as Severity, c as StaticConfigLoader, p as TextClassification, m as TextContent, T as TextNode, U as UserError, V as Validator, W as WrappedError, q as classifyNodeText, P as configPresets, w as definePlugin, r as keywordPatternMatcher, y as ruleExists, t as sliceLocation, s as staticResolver, v as version } from './core.js';
export { d as defineMetadata, m as metadataHelper } from './meta-helper.js';
import 'ajv';
import 'deepmerge';
import './elements.js';
import 'fs';
import '@sidvind/better-ajv-errors';
import './utils/natural-join.js';
import '@babel/code-frame';
import 'kleur';
import '@html-validate/stylish';
import 'semver';
